spec/conformance/index.md

```diff
@@ -77,3 +108,166 @@ completing the launch.
 <br />
 
 [well-known]: ../well-known/index.html
+
+
+### Capability Sets
+
+Additionally, Four *Capability Sets* are defined.  Any individual SMART server will publish a granular list of its capabilities; from this list a client can determine which of these Capability Sets are supported:
+
+#### Patient Access for Standalone Apps
+1. `launch-standalone`
+1. At least one of `client-public` or `client-confidential-symmetric`
+1. `context-standalone-patient`
+1. `permission-patient`
+
+####  Patient Access for EHR Launch (i.e. from Portal)
+1. `launch-ehr`
+1. At least one of `client-public` or `client-confidential-symmetric`
+1. `context-ehr-patient`
+1. `permission-patient`
+
+####  Clinician Access for Standalone
+1. `launch-standalone`
+1. At least one of `client-public` or `client-confidential-symmetric`
+1. `permission-user`
+1. `permission-patient`
+
+####  Clinician Access for EHR Launch
+1. `launch-ehr`
+1. At least one of `client-public` or `client-confidential-symmetric`
+1. `context-ehr-patient` support
+1. `context-ehr-encounter` support
+1. `permission-user`
+1. `permission-patient`
+
+## FHIR Authorization Endpoint and Capabilities Discovery using a FHIR CapabilityStatement
+{:. #using-cs}
+
+### Declaring Support for OAuth2 Endpoints
+
+If a server supports SMART on FHIR authorization for access, it declares support for
+automated discovery of OAuth2 endpoints in its [CapabilityStatement]({{site.data.fhir.path}}/capabilitystatement.html) using the [OAuth Uri extension](#oauth-uris-extension) on the `rest.security` element (or, when using FHIR DSTU2, the
+`Conformance.rest.security` element). Any time a client sees this extension, it
+must be prepared to authorize using SMART's OAuth2-based protocol.
+
+The OAuth extension has the following internal components:
+
+|Component|Conformance Expectation|Type|Description|
```

Comment:
This table looks like it's duplicating content from above. Perhaps we can just cut the table and refer to the one above?